src/components/demo-page/ShoppingCart.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+
+export const ShoppingCart = () => (
+  <div className="shopping-cart">
+    <h1>Shopping Cart</h1>
+
+    <div>
```

Comment:
Can we either get rid of this extra div or give it a `className` to say what it is?

Reply:
Renamed it to shopping-cart-container (I know not too descriptive). But still undecided - I may remove it (will finish cleaning this up with the general css classes re-org)